tox.ini

```diff
@@ -24,15 +24,15 @@ deps =
     dask[array,diagnostics]
     setuptools_scm
 commands =
-    pytest --cov=unyt --cov-append --doctest-modules --doctest-plus --doctest-rst --basetemp={envtmpdir} -W once
+    pytest --cov=unyt --cov-append --doctest-modules --doctest-plus --doctest-rst --basetemp={envtmpdir}
     coverage report --omit='.tox/*'
 
 [testenv:py38-versions]
 deps =
     docutils
     pytest
     sympy==1.5
-    numpy==1.17.5
+    numpy==1.20.0
```

Comment:
I think we can’t bump to such a recent numpy, even if we drop support for python 3.9 we’d still need to support numpy 1.19.

Why did you bump the minimum version? The commit logs talk about `np.concatenate` - couldn’t you put that in an import guard or something similar?

Reply:
`np.concatenate`'s signature changed in numpy 1.20 (the last 2 arguments weren't available before that), so my first implementation for it wasn't compatible with numpy<1.20

I bumped because I was looking for the quicker fix to get CI back to green, but I can refine this now and provide two different implementations, with a runtime check of numpy's version using `packaging` (this would be a new hard dependency, but a cheap one: it's almost certainly an indirect dependency to unyt already)

Reply:
Adding `packaging` is fine.

Reply:
thanks ! updated